analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NonDerivedPrivateClassesShouldBeSealed.CSharp11.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+
+file class ClassWontBeExtended { } // Noncompliant {{File-scoped classes which are not derived in the current assembly should be marked as 'sealed'.}}
```

Comment:
We can use the present tense; we know what the present is, hard to say the future. 🔮

Reply:
Tommorow is a mystery, yesterday is history, today is a gift.
That's why it's called...present 🐼